src/transaction/invoke_function.rs

```diff
@@ -371,7 +371,10 @@ impl InvokeFunction {
             Rc<RefCell<ProgramCache<'_, ClassHash>>>,
         >,
     ) -> Result<TransactionExecutionInfo, TransactionError> {
-        if self.version != Felt252::ONE && self.version != Felt252::ZERO {
+        if !(self.version == Felt252::ZERO
+            || self.version == Felt252::ONE
+            || self.version == Felt252::THREE)
+        {
             return Err(TransactionError::UnsupportedTxVersion(
                 "Invoke".to_string(),
                 self.version,
```

Comment:
The line below contains the valid transaction versions (which are used when displaying the unsupported version error), we should add the new version there too